### DISCUSSION TITLE
Can't install npm packages without error: _execa.default.node is not a function

### DISCUSSION THREAD

**Discussion by dougsmith1000** [gatsbyjs/gatsby#32824] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

Hello,

I've just tried upgrading to Gatsby v3 and am running Gatsby 3.11.0. The initial install works fine, but when I install any other package, I get the following error: 
 
```
 ERROR

UNHANDLED REJECTION _execa.default.node is not a function

  TypeError: _execa.default.node is not a function

  - develop.ts:115 ControllableScript.start
    [frontend]/[gatsby]/src/commands/develop.ts:115:26

  - develop.ts:399 module.exports
    [frontend]/[gatsby]/src/commands/develop.ts:399:18
```

### Reproduction Link

Please use the default starter

### Steps to Reproduce

1. Update/Install gatsby cli to 3.11.0
2. gatsby new [site name]
3. cd [site name]
4. npm install [package]
5. gatsby develop
...

### Expected Result

Development process begins

### Actual Result

See above error

### Environment

```shell
System:
    OS: macOS 11.5.1
    CPU: (12) x64 Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 14.15.4 - ~/.nvm/versions/node/v14.15.4/bin/node
    Yarn: 1.22.10 - ~/.nvm/versions/node/v14.15.4/bin/yarn
    npm: 7.6.0 - ~/.nvm/versions/node/v14.15.4/bin/npm
  Languages:
    Python: 3.7.0 - /usr/local/bin/python
  Browsers:
    Chrome: 92.0.4515.131
    Edge: 92.0.902.73
    Firefox: 90.0.2
    Safari: 14.1.2
  npmGlobalPackages:
    gatsby-cli: 3.11.0
```

**Comment by dougsmith1000** [comment]
I just tested 3.10.0 and everything seems to work. Might just be a 3.11 bug.

**Comment by LekoArts** [comment]
npm might be screwing up your install. Try `npm install` with the `--legacy-peer-deps` flag, too. I followed your steps (but with `yarn` since I used that) and didn't see any problems.

**Reply by dougsmith1000** [reply]
I guess I'll switch careers. Today everything works fine. I ran `--legacy-peer-deps` in one project and now everything works across every new `gatsby new` install without needing to rerun `npm install` with that flag for other packages. Thanks for your help @LekoArts !
